logdna/logdna.py

```diff
@@ -1,72 +1,83 @@
 import sys
 import time
 import json
-from .configs import defaults
 import logging
-import requests
-import threading
 import socket
-from threading import Timer
+import threading
+import requests
```

Comment:
Pythoners (but actually all coders) tend to alphabetize their imports. I wouldn't mind you doing that here.

Reply:
`pylint` was complaining for the order - after fixing the order, didn't care about having alphabetical order that much.